src/windowmanagement/HelpMenu.tsx

```diff
@@ -1,26 +1,97 @@
 import { Button, Menu } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+
+import {
+  useGetBoolPropertyValue,
+  useGetIntPropertyValue,
+  useGetStringPropertyValue,
+  useOpenSpaceApi
+} from '@/api/hooks';
+import { About } from '@/components/About/About';
+import {
+  BookIcon,
+  ConsoleIcon,
+  ExitAppIcon,
+  FeedbackIcon,
+  InformationCircleOutlineIcon,
+  OpenInBrowserIcon,
+  RouteIcon
+} from '@/icons/icons';
 
 export function HelpMenu() {
+  const luaApi = useOpenSpaceApi();
+  const [opened, { open, close }] = useDisclosure(false);
+  const [isConsoleVisible, setIsConsoleVisible] =
+    useGetBoolPropertyValue('LuaConsole.IsVisible');
+  const [portProperty] = useGetIntPropertyValue('Modules.WebGui.Port');
+  const [addressProperty] = useGetStringPropertyValue('Modules.WebGui.Address');
+
+  function toggleLuaConsole() {
+    setIsConsoleVisible(!isConsoleVisible);
+  }
+
+  function openGuiInBrowser() {
+    const port = portProperty ?? 4670; // TODO: Change this to correct port when hosted
```

Comment:
This could already be set to the correct port, since we get it form the property anyways right now. Gives us one less thing to care about later on

```suggestion
    const port = portProperty ?? 4680;
```